solution-base/build.sh

```diff
@@ -194,7 +199,11 @@ function render_mongodb_sharded_yamls()
         --set 'configsvr.persistence.selector.matchLabels.app\.kubernetes\.io/name=mongodb-sharded-config' \
         --set 'configsvr.persistence.selector.matchLabels.app\.kubernetes\.io/part-of=zenko' \
         --set "configsvr.mongodbExtraFlags={--setParameter rollbackTimeLimitSecs=259200}" \
-        --set "shardsvr.dataNode.mongodbExtraFlags={--setParameter rollbackTimeLimitSecs=259200}" \
+        --set mongos.resources.requests.memory=${MONGODB_MONGOS_RAM_REQUEST} \
+        --set mongos.resources.limits.memory=${MONGODB_MONGOS_RAM_LIMIT} \
+        --set "shardsvr.dataNode.mongodbExtraFlags={--wiredTigerCacheSizeGB=${MONGODB_WIREDTIGER_CACHE_SIZE_GB}}" \
```

Comment:
for these flags we need a variable indeed, but i wonder if it should be specifically `MONGODB_WIREDTIGER_CACHE_SIZE_GB`, or a more general variable like `MONGODB_EXTRA_FLAGS` ?